### PR TITLE
Release 4.1.2.0

### PR DESCRIPTION
**Pull request by cdepillabout**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ more deeply nested.  It would be even more difficult to read.
 
 ![example screenshot](https://raw.githubusercontent.com/cdepillabout/pretty-simple/master/img/pretty-simple-example-screenshot.png)
 
+There's a [web app](https://cdepillabout.github.io/pretty-simple) compiled with
+GHCJS where you can play around with `pretty-simple` running in your browser.
```
Comment (cdepillabout) [inline README.md:43]: @georgefst I added this to the README, but please feel free to send a follow up PR or something if you think there is a better way to show it off.  I think it is really neat, and I bet potential users might really want to play with it.

It is also a useful tool in its own right, similar to the `pretty-simple` CLI tool.

**Comment by cdepillabout** [comment]
Released to Hackage: https://hackage.haskell.org/package/pretty-simple-4.1.2.0

File: README.md (hunk 1 above)
Comment (georgefst) [reply README.md:43]: Cool, yeah I've been meaning to do this, as well as adding a link back to Hackage/GitHub from the web app.

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -1,10 +1,7 @@
-# TODO use this trigger once tested
-# on:
-#   push:
-#     branches:
-#       - master
 on:
-  pull_request:
+  push:
+    branches:
+      - master
```
Comment (georgefst) [inline .github/workflows/deploy.yaml:4]: Whoops. thanks, these should obviously have been uncommented before that PR was merged.

Comment (cdepillabout) [reply .github/workflows/deploy.yaml:4]: No problem!  All good now :-)

**Pull request merged**